gapis/api/cmd_id_group.go

```diff
@@ -372,8 +374,20 @@ func (g *CmdIDGroup) AddRoot(rootidx []uint64) *SubCmdRoot {
 		switch first := g.Spans[s].(type) {
 		case *CmdIDGroup:
 			return first.AddRoot(rootidx)
+		case *CmdIDRange:
+			firstHalf := &CmdIDRange{first.Start, first.End}
```

Comment:
oops, I was just trying to use copy-constructor to build a CmdIDRange. I will fix this.